ubyssey/static/src/styles/modules/events/_calendar.scss

```diff
@@ -0,0 +1,178 @@
+$border                           : 1px dotted $color-border-gray-dark;
+$calendar-event-time-label-color  : #006fcc;
+
+$tiles-margin                     : 35px;
+
+#calendar {
+  // max-width: 1600px;
+  min-height: 500px;
+  margin-bottom: 40px;
+
+  @media($bp-smaller-than-phablet) {
+    padding: 0 2px;
+
+    h1 {
+       padding: 0 5px;
+    }
+  }
+}
+
+.calendar-container {
```

Comment:
Each classname should follow the conventions from the [style guide](https://github.com/ubyssey/ubyssey-dispatch-theme/wiki/SASS-Style-Guide):

```scss
.c-event-calendar__container {
  // Structure
  width: 100%;
}
```